bazel/patches/datadog_agent_toolchain.patch

```diff
@@ -0,0 +1,513 @@
+                 flag_groups = [
+                     flag_group(
+                         flags = ["%{user_compile_flags}"],
+@@ -439,17 +391,6 @@ def _impl(ctx):
+         ] if len(extra_cxxflags) > 0 else [],
+     )
+ 
+-    extra_fflags_feature = feature(
+-        name = "extra_fflags",
+-        enabled = True,
+-        flag_sets = [
+-            flag_set(
+-                actions = [FORTRAN_ACTION_NAMES.fortran_compile],
+-                flag_groups = [flag_group(flags = extra_fflags)],
+-            ),
+-        ] if len(extra_fflags) > 0 else [],
+-    )
+-
+     extra_ldflags_feature = feature(
+         name = "extra_ldflags",
+         enabled = True,
+@@ -476,7 +417,6 @@ def _impl(ctx):
+                     ACTION_NAMES.cpp_module_codegen,
+                     ACTION_NAMES.lto_backend,
+                     ACTION_NAMES.clif_match,
+-                    FORTRAN_ACTION_NAMES.fortran_compile,
+                 ] + all_link_actions + lto_index_actions,
+                 flag_groups = [
+                     flag_group(
+@@ -489,9 +429,6 @@ def _impl(ctx):
+     )
+ 
+     features = sanitizers_features + [
+-        fortran_compile_flags_feature,
+-        static_libgfortran_feature,
+-        fortran_link_flags_feature,
+         default_compile_flags_feature,
+         include_paths_feature,
+         library_search_directories_feature,
+@@ -507,7 +444,6 @@ def _impl(ctx):
+         redacted_dates_feature,
+         extra_cflags_feature,
+         extra_cxxflags_feature,
+-        extra_fflags_feature,
+         extra_ldflags_feature,
+     ]
+ 
+@@ -541,7 +477,6 @@ cc_toolchain_config = rule(
+         "cxx_builtin_include_directories": attr.string_list(mandatory = True),
+         "extra_cflags": attr.string_list(mandatory = True),
+         "extra_cxxflags": attr.string_list(mandatory = True),
+-        "extra_fflags": attr.string_list(mandatory = True),
+         "extra_ldflags": attr.string_list(mandatory = True),
+         "tool_paths": attr.string_dict(mandatory = True),
+     },
+@@ -557,7 +492,7 @@ def _sanitizer_feature(sanitizer):
+         name = sanitizer.name,
+         flag_sets = [
+             flag_set(
+-                actions = all_compile_actions + [FORTRAN_ACTION_NAMES.fortran_compile],
++                actions = all_compile_actions,
+                 flag_groups = [
+                     flag_group(
+                         flags = sanitizer.cflags,
+diff --git a/toolchain/defs.bzl b/toolchain/defs.bzl
+index 00ef405..94d2c49 100644
+--- a/toolchain/defs.bzl
++++ b/toolchain/defs.bzl
+@@ -32,6 +32,7 @@ def _gcc_toolchain_impl(rctx):
+     absolute_toolchain_root = str(rctx.path("."))
+     execroot = paths.normalize(paths.join(absolute_toolchain_root, "..", ".."))
+     toolchain_root = paths.relativize(absolute_toolchain_root, execroot)
++    rctx.symlink("{}/{}-unknown-linux-gnu/bin".format(toolchain_root, rctx.attr.target_arch), "{}/xbin".format(toolchain_root))
```

Comment:
I'm not a big fan of comments because they frequently paraphrase the next few lines of code (and later diverge from them), but I would love to have one here that explains the purpose of the symlink.

Reply:
That's not needed actually, removed